app/controllers/ingredient_dishes_controller.rb

```diff
@@ -0,0 +1,8 @@
+class IngredientDishesController < ApplicationController
+  def create
+    dish = Dish.find(params[:id])
+    ingredient = Ingredient.find(params[:ingredient_id])
+    IngredientDish.create(ingredient: ingredient, dish: dish)
+    redirect_to "/dishes/#{dish.id}"
+  end
+end
```

Comment:
Great job with this, this is 100% the correct action and controller that should be handling this request.